src/main/scala/model/WaveManager.scala

```diff
@@ -50,4 +50,4 @@ object Generator:
     private def createEnemyList(n: Int)(l: List[Enemy]): List[Enemy] =
       n match
         case 0 => l
-        case _ => createEnemyList(n - 1)(l = l :+ new Zombie((Random.between(0, NumOfLanes), LanesLength + Random.between(0, 20))))
\ No newline at end of file
+        case _ => createEnemyList(n - 1)(l = l :+ FastZombie())
```

Comment:
I suppose this was made for testing purposes.
Revert the changes and make the wave generator spawn only basic zombies.